### PR TITLE
added lowest-common-ancestor

### PR DESCRIPTION
**Pull request by AdilMah**

### CODE REVIEW

**Comment by prateekiiest** [comment]
document the code

File: BInary_Tree/level_order_traversal.c [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#include<stdlib.h>
+////////////////////////////////////////////////////
+typedef struct Node
+{
+	int data;
+	struct Node* left;
+	struct Node* right;
+
+}T;
+///////////////////////////////////////////////////
```
Comment (prateekiiest) [inline BInary_Tree/level_order_traversal.c]: remove this 
its unnecessary here

File: BInary_Tree/level_order_traversal.c [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+T* newnode(int);
+void LevelOrder(T*);
+T* printGivenLevel(T*,int);
+int height(T*);
+///////////////////////////////////////////////////
```
Comment (prateekiiest) [inline BInary_Tree/level_order_traversal.c]: Also here and all other places

Document the code

File: BInary_Tree/lowest_common_ancestor.c [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+#include<stdlib.h>
+
+typedef struct Node
+{
+	int data;
+	struct Node* left;
+	struct Node* right;
+
+}T;
+T* newnode(int );
```
Comment (prateekiiest) [inline BInary_Tree/lowest_common_ancestor.c]: some space here needed

File: BInary_Tree/lowest_common_ancestor.c [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ 	temp->left=NULL;
+ 	temp->right=NULL;
+ 	return temp;
+
+}>left&&findpath(root,path1,n))||(root->right && findpath(root,path2,int n)))
+						return true;
+
+	path.pop_back();
+	return false;
+}
```
Comment (prateekiiest) [inline BInary_Tree/lowest_common_ancestor.c]: document all code

File: BInary_Tree/level_order_traversal.c (hunk 1 above)
Comment (prateekiiest) [reply BInary_Tree/level_order_traversal.c]: ///////////////////////////  remove them

**Comment by prateekiiest** [comment]
codacy fails, see the details

**Comment by AdilMah** [comment]
deleted the two files, polynomial addition and LCA, would add them later. Updated level order traversal program

**Review by prateekiiest — APPROVED** [review]

**Comment by prateekiiest** [comment]
Thanks @AdilMah 
Abhi semesters ke liye paro ;p

**Comment by AdilMah** [comment]
:) sure bhaiyya

**Comment by prateekiiest** [comment]
I hope you got to understand how open source works and contributions in program like GSoc occurs

**Pull request merged**